README.md

```diff
@@ -16,8 +16,7 @@ from real-time workload traffic seamlessly.
 
 ## Solution
```

Comment:
Quick comment: The Chart uses the branch gh-pages to update the helm yaml repo: https://github.com/apiclarity/apiclarity/blob/gh-pages/index.yaml
However there is also the README.md that can be seen here: https://apiclarity.github.io/apiclarity/ 

Two solutions:
- move the README.md changes also on the gh-pages branch
- simplify the README.md on gh-pages and keep something standard like: "This is the HELM Repo ..."

Reply:
Thanks for that, I will update the `README.md` in gh-pages branch once we will merge this one